activerecord/test/cases/test_fixtures_test.rb

```diff
@@ -70,5 +70,133 @@ def test_run_successfully
       clean_up_connection_handler
       FileUtils.rm_r(tmp_dir)
     end
+
+    def test_transactional_tests_per_db_explicitly_disabled
+      tmp_dir = Dir.mktmpdir
+      File.write(File.join(tmp_dir, "zines.yml"), <<~YML)
+      going_out:
+        title: Hello
+      YML
+
+      klass = Class.new(Minitest::Test) do
+        include ActiveRecord::TestFixtures
+
+        self.fixture_paths = [tmp_dir]
+        self.use_transactional_tests = true
+        self.skip_transactional_tests_for_database :primary
+
+        fixtures :all
+
+        def test_run_successfully
+          assert_equal("Hello", Zine.first.title)
+          assert_equal("Hello", zines(:going_out).title)
+          # Change the data in the primary connection
+          Zine.first.update!(title: "Goodbye")
+        end
+      end
+
+      test_result = klass.new("test_run_successfully").run
+      assert_predicate(test_result, :passed?)
+      # Ensure that the primary connection was NOT rolled back
+      assert_equal("Goodbye", Zine.first.title)
+    ensure
+      FileUtils.rm_r(tmp_dir)
+    end
+
+    def test_transactional_tests_per_db_explicitly_enabled
+      tmp_dir = Dir.mktmpdir
+      File.write(File.join(tmp_dir, "zines.yml"), <<~YML)
+      going_out:
+        title: Hello
+      YML
+
+      klass = Class.new(Minitest::Test) do
+        include ActiveRecord::TestFixtures
+
+        self.fixture_paths = [tmp_dir]
+        self.use_transactional_tests = false
+        self.use_transactional_tests_for_database :primary
```

Comment:
@mjcheetham  @matthewd  shouldn't this method ( `use_transactional_tests_for_database` ) be added in the CHANGELOG as well. something like this : 
```ruby
    class TransactionalForSharedOnlylTest < ActiveSupport::TestCase
      self.use_transactional_tests = false
      use_transactional_tests_for_database :shared
    end
```

Reply:
No; the changelog is a summary of the change (it's rare to include a code sample at all). It is not the API documentation.

Reply:
Ok about the code sample. But shouldn't the CHANGELOG at least mention this other public configuration method I mentioned, this way the user can go and read the rdoc documentation related to it. 

Otherwise how would anyone guess that this method was added as part of the next release